src/sound.ts

```diff
@@ -1,59 +1,87 @@
 import throttle from 'lodash-es/throttle'
-import { SoundEffect } from 'capacitor-sound-effect'
-import { DeviceInfo } from '@capacitor/device'
 import settings from './settings'
 import { isForeground } from './utils/appMode'
 
-function shouldPlay(): boolean {
-  return settings.general.sound() && isForeground()
-}
+let ctx = typeof(window.AudioContext) != 'undefined' ? new AudioContext() : null
 
 export default {
-  load(info: DeviceInfo): Promise<void> {
-    const ext = info.platform === 'ios' ? '.aifc' : '.mp3'
-
-    return Promise.all([
-      SoundEffect.loadSound({ id: 'move', path: `sounds/move${ext}` }),
-      SoundEffect.loadSound({ id: 'capture', path: `sounds/capture${ext}` }),
-      SoundEffect.loadSound({ id: 'explosion', path: `sounds/explosion${ext}` }),
-      SoundEffect.loadSound({ id: 'lowtime', path: `sounds/lowtime${ext}` }),
-      SoundEffect.loadSound({ id: 'dong', path: `sounds/dong${ext}` }),
-      SoundEffect.loadSound({ id: 'berserk', path: `sounds/berserk${ext}` }),
-      SoundEffect.loadSound({ id: 'clock', path: `sounds/clock${ext}` }),
-      SoundEffect.loadSound({ id: 'confirmation', path: `sounds/confirmation${ext}` }),
-    ]).then(() => { /* noop */ })
-  },
-  move() {
-    if (shouldPlay()) SoundEffect.play({ id: 'move' })
+  load(): Promise<void> {
+    return ctx == null 
+      ? Promise.resolve()
+      : Promise.all([
+        loadSound('move').then(buf => (move = buf)),
+        loadSound('capture').then(buf => (capture = buf)),
+        loadSound('explosion').then(buf => (explosion = buf)),
+        loadSound('lowtime').then(buf => (lowtime = buf)),
+        loadSound('dong').then(buf => (dong = buf)),
+        loadSound('berserk').then(buf => (berserk = buf)),
+        loadSound('clock').then(buf => (clock = buf)),
+        loadSound('confirmation').then(buf => (confirmation = buf)),
+      ]).then(() => console.log('all sounds loaded.'))
+  },
+  
+  resume(): void {
+    if (ctx != null ) {  
```

Comment:
don't understand this. why not use `AudioContext.suspend()` and `AudioContext.resume()` instead?